releng/releng-target/lsp4j.target.target

```diff
@@ -13,5 +13,9 @@
 			<unit id="javax.websocket" version="0.0.0"/>
 			<unit id="javax.websocket.source" version="0.0.0"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="false" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/cbi/updates/license/"/>
+			<unit id="org.eclipse.license.feature.group" version="2.0.2.v20181016-2210"/>
```

Comment:
Minor change needed - this should be 0.0.0 version so that the latest version of the license is automatically picked up.

e.g: https://github.com/eclipse-cdt/cdt/blob/79b1f157dd9f51a535f23f7930ae3518feb526e3/releng/org.eclipse.cdt.target/cdt.target#L6

Reply:
I am sorry, but would not that mean, in theory at least, that a new EPL version like 2.1.x or even 3.x might be inadvertently picked up? Would not it be safer to err on the side of specifying the exact version?

Reply:
The `0.0.0` is just standard practice across multiple Eclipse projects, it ensures that latest version is picked up. It does mean that it will become EPL3 automatically, but almost all project do it this way. I assume that it is to make sure we get the latest 2.0.x version. If target files allowed ranges (like [tpd](https://github.com/eclipse-cbi/targetplatform-dsl)) then the problem would be better solved. 

Of course, the EPL 2 license feature hasn't been updated in ~3 years now, so probably not a big issue.

BTW https://bugs.eclipse.org/bugs/show_bug.cgi?id=566576 was supposed to have documented this, but I guess it never happened.

Reply:
I mentioned it only because in Handly we used the same approach as suggested by Mat to err on the side of caution with the legal stuff: https://git.eclipse.org/c/handly/org.eclipse.handly.git/tree/targets/base/base.target#n23. The downside is that the license version has to manually updated when a new revision becomes available and there might be no notification about it. The upside is that we don't run the risk of inadvertently picking up an inappropriate license.

Thanks for the reference to the bug. Although some links are broken, the last sentence is "If there is a new version of the license we update the version of the license feature we include in the target platform", which seems to indicate that the version is assumed to be manually maintained.

Having said that, I understand that specifying the `0.0.0` is a common practice across multiple Eclipse projects.

Reply:
It was a minor thing - the fact that other do it differently (like handly) means that we don't need to change this. Resolving.